JITpackage/Source/grb_jitpackage.c

```diff
@@ -112,14 +113,61 @@ bool match_prefix (char *string, char *prefix)
 int main (int argc, char **argv)
 {
 
+    //--------------------------------------------------------------------------
+    // get list of files to be processed
+    //--------------------------------------------------------------------------
+
+    char **file_list = NULL;
+    size_t nfiles = 0;
+
+    if (argc == 2 && argv[1][0] == '@')
+    {
+        // input argument is a "response file" containing the file list
+
+        // open file
+        FILE *fr = fopen (argv[1]+1, "rb") ;
+        OK (fr != NULL) ;
+
+        // get number of lines in file
+        char ch;
+        do
+        {
+            ch = fgetc (fr);
+            if (ch == '\n')
+                nfiles++;
+        } while (ch != EOF);
+
+        // read file list from response file
+        rewind (fr);
+        file_list = malloc ( (nfiles+1) * sizeof (file_list) );
+        // prepend empty element for compatibility with argv
+        file_list[0] = malloc (1);
+        file_list[0][0] = '\0';
+        char temp[200];
```

Comment:
200 is very short for a maximum path length

Reply:
Good point. That is short. I can fix that.

Reply:
Is setting this to the value of `MAX_PATH` as defined by glibc ok? That is 4096. 
See #305.